samples/react-kiota-custom-api-client/teamified-client/src/webparts/teamsList/client/teamifiedApiClient.ts

```diff
@@ -0,0 +1,52 @@
+import {PingRequestBuilder} from './ping/pingRequestBuilder';
+import {TeamsItemRequestBuilder} from './teams/item/teamsItemRequestBuilder';
+import {TeamsRequestBuilder} from './teams/teamsRequestBuilder';
+import {enableBackingStoreForSerializationWriterFactory, getPathParameters, ParseNodeFactoryRegistry, registerDefaultDeserializer, registerDefaultSerializer, RequestAdapter, SerializationWriterFactoryRegistry} from '@microsoft/kiota-abstractions';
+import {JsonParseNodeFactory, JsonSerializationWriterFactory} from '@microsoft/kiota-serialization-json';
+import {TextParseNodeFactory, TextSerializationWriterFactory} from '@microsoft/kiota-serialization-text';
+
+/** The main entry point of the SDK, exposes the configuration and the fluent API. */
+export class TeamifiedApiClient {
+    /** Path parameters for the request */
+    private readonly pathParameters: Record<string, unknown>;
+    /** The ping property */
+    public get ping(): PingRequestBuilder {
+        return new PingRequestBuilder(this.pathParameters, this.requestAdapter);
+    }
+    /** The request adapter to use to execute the requests. */
+    private readonly requestAdapter: RequestAdapter;
+    /** The teams property */
+    public get teams(): TeamsRequestBuilder {
+        return new TeamsRequestBuilder(this.pathParameters, this.requestAdapter);
+    }
+    /** Url template to use to build the URL for the current request builder */
+    private readonly urlTemplate: string;
+    /**
+     * Instantiates a new TeamifiedApiClient and sets the default values.
+     * @param requestAdapter The request adapter to use to execute the requests.
+     */
+    public constructor(requestAdapter: RequestAdapter) {
+        if(!requestAdapter) throw new Error("requestAdapter cannot be undefined");
+        this.pathParameters = {};
+        this.urlTemplate = "{+baseurl}";
+        this.requestAdapter = requestAdapter;
+        registerDefaultSerializer(JsonSerializationWriterFactory);
```

Comment:
Hi @luismanez, the backend API project is working as expected. However, I'm having trouble building and running the web part. I'm getting exceptions with teamifiedApiClient.ts generated file when registering serializers and deserializers. Is it something that you are experiencing on your environment, too? Thanks.

Reply:
Hi @PaoloPia haven´t experienced that issue. What you mean by "registering serializers and deserializers"? cos I didn´t register anything. If I have to say something I´d say is a package version issue, but please, provide more info about the error message. Thanks!

Reply:
@luismanez the issue is in the code highlighted just above my comment. Lines from 33 to 36 of file teamifiedApiClient.ts have registration instructions that don't work because of a type definition mismatch. Thanks.

Reply:
Thanks @PaoloPia I was able to reproduce it in another machine. As suspected, the issue was related with Kiota packages versions. Kiota is still in preview and having breaking changes often. The Kiota abstractions package is now in preview7, and my code was generated with preview6. I've updated the package.json file to force the exact match version. I've removed the node_modules and nom install, and now is working fine. Can you give it a try and let me know if it fixes the issue for you too?
Thanks!

Reply:
Thanks @luismanez! Now it works. I'm going to merge it!